### PR TITLE
Adding default_ports option

### PR DESCRIPTION
**Pull request by dbpolito**
Default Port forwarding makes hard to use multiple homestead instances at the same time. With new introduced option, now we're able to simply do not forward any port at all, using direct VM IP instead of port forwards.

With new introduced option, it only disables if `default_ports: false`

### CODE REVIEW

File: scripts/homestead.rb [hunk 1]
```diff
@@ -40,7 +40,7 @@ def Homestead.configure(config, settings)
     # Configure A Few VMware Settings
     ["vmware_fusion", "vmware_workstation"].each do |vmware|
       config.vm.provider vmware do |v|
-        v.vmx["displayName"] = "homestead"
+        v.vmx["displayName"] = settings["name"] ||= "homestead-7"
```
Comment (taylorotwell) [inline scripts/homestead.rb]: Why this change? Default homestead is already on PHP 7.

Comment (dbpolito) [inline scripts/homestead.rb]: I just added the same logic from virtual box provider:

https://github.com/laravel/homestead/blob/master/scripts/homestead.rb#L32

Comment (taylorotwell) [inline scripts/homestead.rb]: Hmm, ok :)

**Comment by taylorotwell** [comment]
So is this different than just removing the ports from the configuration?

**Comment by dbpolito** [comment]
Yes because if you just remove the ports from config file the default_ports are going to be added anyway...

My initial idea was just, if there is a `port` config and it's a empty array, don't add default ports at all... Just add default ports if the `port` config is commented out, which is the default.

**Comment by taylorotwell** [comment]
Personally I think this should maybe be called port_forwarding: false / true instead of default_ports: false

**Comment by dbpolito** [comment]
In my opinion `default_ports` makes more sense because if you add ports to your config, and set `default_ports: false`, your custom ports are still going to be added...

`port_forwarding: false` makes me think no ports will be added at all... But i'm totally ok with that change...
